### DISCUSSION TITLE
@rollup/plugin-commonjs  Cannot read properties of undefined (reading '_container')

### DISCUSSION THREAD

**Discussion by xiaoqianghaochi** [rollup/plugins#1803] [Q&A]
- Rollup Plugin Name: @rollup/plugin-commonjs
- Rollup Plugin Version:28.0.1
- Rollup Version:4.24.0
- Operating System (or Browser):
- Node Version: mac os
- Link to reproduction _(⚠️ read below)_: https://stackblitz.com/edit/rollup-repro-1hn9af?file=package.json

TypeError: Cannot read properties of undefined (reading '_container')
    at load (file:///Users/*/vite-ssr-test/node_modules/vite/dist/node/chunks/dep-Cyk9bIUq.js:49176:16)

### Expected Behavior
SSR loads commonjs normally

### Actual Behavior

TypeError: Cannot read properties of undefined (reading '_container')
    at load (file:///Users/*/vite-ssr-test/node_modules/vite/dist/node/chunks/dep-Cyk9bIUq.js:49176:16)

![image](https://github.com/user-attachments/assets/dcb5f89f-561c-45b9-a172-bbc393eff5ec)

![image](https://github.com/user-attachments/assets/7e2b41a3-924c-4fd9-9a4e-908999a7c95c)

![image](https://github.com/user-attachments/assets/4cdfdd3f-afd1-443f-b90d-f47deb26b70d)

Reason: This was lost after passing the method

**Comment by xiaoqianghaochi** [comment]
@cbenz @rhengles @flipsasser @eventualbuddha Hi, could you help me with this problem?

**Reply by cbenz** [reply]
I feel I'm not the targeted person :smile:

**Reply by xiaoqianghaochi** [reply]
OK，be sorry ， thank you all the same.

**Comment by hlg-facai** [comment]
Doesn't anyone care about this?
